### DISCUSSION TITLE
Thought I would try it but with a twist.

### DISCUSSION THREAD

**Discussion by barkby** [martinwoodward/octolamp#3] [Show and tell]
Thought I would do something different as I did not have any white PLA.
I decided to lasercut the logo, insert the LEDs with the frame and then fill that gap with resin and white pigment.
1 layer of 3mm ply for back
3 layers of 3mm ply for the logo all stacked

Still a work in progress and I need to clean up and add an additional layer or more Opaque White Resin to defuse then LEDs more.

![20230119_164206](https://user-images.githubusercontent.com/9532453/213515415-4ebf6e64-5f02-4d39-91d4-b3313cd06820.jpg)

![20230119_172351](https://user-images.githubusercontent.com/9532453/213516279-9e3e45ba-d56e-4d77-9092-e23d926dac47.jpg)

**Comment by martinwoodward** [comment]
Oh wow!  That makes for a much thinner build!!

**Comment by mertbozkir** [comment]
Any progress? Have you able to try with opaque white resin?
